src/applications/personalization/dashboard/manifest.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  appName: 'Dashboard',
+  entryFile: './dashboard-entry.jsx',
+  entryName: 'dashboard',
+  receiveContentProps() {
+    // pulled from isBrandConsolidationEnabled helper
+    // manifest is used during node build which is not babel transformed-
+    // this prevents importing the helper
+    this.rootUrl = window.settings && window.settings.brandConsolidationEnabled ? '/my-va' : '/dashboard';
```

Comment:
`receiveContentProps` is a little [too magical](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13513), but it actually provides the page path of the content page containing the entryname `dashboard`.  So, if you move the Dashboard's content file around in `content`, you can bind to that page path the same way. Then, during brand-consolidation builds, it will be seamless, because the page path of the content file in the `va-gov` directory will be provided instead.

But anyway, you should be able to borrow the same function as [Letters](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/letters/manifest.js) and see that `path` argument reflect the Dashboard's URL, and update automatically based on Vets.gov or brand-consolidation builds. Letters moved from `/download-va-letters/` to `/records/download-va-letters/` and that was the only update it needed to make.